package.json

```diff
@@ -38,6 +38,10 @@
   },
   "dependencies": {
     "@bubblewrap/core": "^1.4.0",
+    "@types/cli-progress": "^3.7.0",
+    "@types/mime-types": "^2.1.0",
+    "cli-progress": "^3.8.2",
+    "jimp": "^0.14.0",
```

Comment:
Why are we adding jimp in this CL?

Reply:
I'm not sure, I didn't touch in my last work

Reply:
Can you double-check if those dependencies are necessary here? This is the root `package.json`. It seems to me those dependencies should be in `/packages/core/package.json` or `/packages/cli/package.json`.